a3fe/configuration/_engine_runner_config.py

```diff
@@ -2,81 +2,43 @@
 
 from __future__ import annotations
 
-import copy as _copy
-import logging as _logging
 import yaml as _yaml
-from abc import ABC, abstractmethod
+from pydantic import BaseModel as _BaseModel
 from typing import Any as _Any
 from typing import Dict as _Dict
 
-from ..run._logging_formatters import _A3feStreamFormatter
+class EngineRunnerConfig(_BaseModel):
```

Comment:
Thanks, you're right to make this a base model, but please also keep it as an ABC, because this allows us to make a template which is enforced. Thanks